src/librustdoc/html/static/js/main.js

```diff
@@ -951,13 +951,14 @@ function hideThemeButtonState() {
             "Accepted kinds are: <code>fn</code>, <code>mod</code>, <code>struct</code>, \
              <code>enum</code>, <code>trait</code>, <code>type</code>, <code>macro</code>, \
              and <code>const</code>.",
-            "Search functions by type signature (e.g., <code>vec -&gt; usize</code> or \
+            "Search functions by type signature (e.g., <code>-&gt; usize</code> or \
```

Comment:
Why did this change?

Reply:
Because before you needed to have a "name" before the `->` symbol. It's not the case anymore (you can still have a name, it's simply not mandatory).

Reply:
> Because before you needed to have a "name" before the `->` symbol.

I don't think that's the case. [`-> usize`](https://doc.rust-lang.org/nightly/std/?search=-%3E%20usize) works.

Reply:
> I don't think that's the case. [`-> usize`](https://doc.rust-lang.org/nightly/std/?search=-%3E%20usize) works.

Oh indeed.

> Also, I think it's confusing to have the example be `-> usize` since it looks weird.

I don't really know how to document this efficiently... Maybe we should rewrite the help popup a bit more?